components/scream/tests/coupled/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 # NOTE: if you have baseline-type tests, add the subdirectory OUTSIDE the following if statement
 if (NOT ${SCREAM_BASELINES_ONLY})
-  if ("${SCREAM_DYNAMICS_DYCORE}" STREQUAL "HOMME")
-    add_subdirectory(homme_physics)
-  endif()
   if (SCREAM_DOUBLE_PRECISION)
-    add_subdirectory(shoc_cld_p3_rrtmgp)
+    add_subdirectory(shoc_cld_spa_p3_rrtmgp)
```

Comment:
We should have probably called this test/folder `scream_physics` or `full_physics`, so it would have stayed the same, despite us adding something every time. If you feel like changing it, I'd be ok with that. Otherwise, nvm.